CheckersClient/src/main/java/cst/Checkers.java

```diff
@@ -1,39 +1,210 @@
+   static int playerOne7x = 0;
+   static int playerOne7y = 0;
+   static int playerOne8x = 0;
+   static int playerOne8y = 0;
+   static int playerOne9x = 0;
+   static int playerOne9y = 0;
+   static int playerOne10x = 0;
+   static int playerOne10y = 0;
+   static int playerOne11x = 0;
+   static int playerOne11y = 0;
+   static int playerOne12x = 0;
+   static int playerOne12y = 0;
+   static int playerTwo1x = 0;
+   static int playerTwo1y = 0;
+   static int playerTwo2x = 0;
+   static int playerTwo2y = 0;
+   static int playerTwo3x = 0;
+   static int playerTwo3y = 0;
+   static int playerTwo4x = 0;
+   static int playerTwo4y = 0;
+   static int playerTwo5x = 0;
+   static int playerTwo5y = 0;
+   static int playerTwo6x = 0;
+   static int playerTwo6y = 0;
+   static int playerTwo7x = 0;
+   static int playerTwo7y = 0;
+   static int playerTwo8x = 0;
+   static int playerTwo8y = 0;
+   static int playerTwo9x = 0;
+   static int playerTwo9y = 0;
+   static int playerTwo10x = 0;
+   static int playerTwo10y = 0;
+   static int playerTwo11x = 0;
+   static int playerTwo11y = 0;
+   static int playerTwo12x = 0;
+   static int playerTwo12y = 0;
+   static int newGame = 0;
 
     /**
      * Default constructor. 
      */
-    public Checkers() {
-        // TODO Auto-generated constructor stub
+    public Checkers() 
+    {
+        super();
     }
 
-	/**
-	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
-	 */
-	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-	   request.getRequestDispatcher("checkers.jsp").forward(request, response);
-		// TODO Auto-generated method stub
-		// response.getWriter().append("Served at: ").append(request.getContextPath());
-	}
-
-	/**
-	 * @see HttpServlet#doPost(HttpServletRequest request, HttpServletResponse response)
-	 */
-	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-		// TODO Auto-generated method stub
-		doGet(request, response);
-	}
+   /**
+    * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
+    */
+   protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+      request.getRequestDispatcher("checkers.jsp").forward(request, response);
+      response.getWriter().append("Served at: ").append(request.getContextPath());
+   }
 
+   /**
+    * @see HttpServlet#doPost(HttpServletRequest request, HttpServletResponse response)
+    */
+   protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+      HttpSession session = request.getSession(true);
+      Integer counter = (Integer)session.getAttribute("counter");
+      if (counter == null || newGame == 0) 
+      {
+         
+         counter = new Integer(0);
+         counter++;
+         playerOne1x = 75;
```

Comment:
that would be helpful, I want to add something to keep track of each spot, so the hardcoded can be replaced by variables. Will work on it.